<?xml version="1.0" ?>
<launch>    
  <arg name="with_gripper"	   default="false" />
  <param name="gazebo"                   value="true"     type="bool"/>
  <param name="gazebo_robot_name"        value="open_manipulator_pro" />
  <param name="offset_table"             value="$(find open_manipulator_pro_manager)/config/offset.yaml"/>

  <group if="$(arg with_gripper)">
    <param name="robot_file_path"          value="$(find open_manipulator_pro_manager)/config/open_manipulator_pro_with_gripper.robot"/>
    <param name="init_file_path"           value="$(find open_manipulator_pro_manager)/config/open_manipulator_pro_with_gripper_dxl_init.yaml"/>
  </group>
  <group unless="$(arg with_gripper)">
    <param name="robot_file_path"          value="$(find open_manipulator_pro_manager)/config/open_manipulator_pro.robot"/>
    <param name="init_file_path"           value="$(find open_manipulator_pro_manager)/config/open_manipulator_pro_dxl_init.yaml"/>
  </group>
  <!-- Manipulator Manager -->
  <node pkg="open_manipulator_pro_manager" type="open_manipulator_pro_manager" name="open_manipulator_pro_manager" output="screen"/>
</launch>

